src/components/Molecules/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Wrapper,
+  InnerWrapper,
+  Form,
+  SearchWrapper,
+  SearchField,
+  ActionWrapper,
+  ActionButton
+} from './SearchInput.style';
+
+const SearchInput = ({ onChangeInput, placeholder }) => {
+  return (
+    <Wrapper>
+      <InnerWrapper>
+        <Form>
+          <SearchWrapper>
+            <SearchField
+              onChange={onChangeInput}
+              name="search"
+              showLabel={false}
+              type="text"
+              label="Search"
+              id="search"
+              placeholder={placeholder}
+              errorMsg=""
+            />
+          </SearchWrapper>
+          <ActionWrapper>
+            <ActionButton
+              type="submit"
+              id=""
+              name="action"
+              placeholder=""
+              label=""
+              errorMsg=""
+            />
+          </ActionWrapper>
+        </Form>
+      </InnerWrapper>
+    </Wrapper>
+  );
+};
+
+SearchInput.propTypes = {
+  onChangeInput: PropTypes.func.isRequired,
```

Comment:
can we rename this to just onChange for the sake of consistency between all inputs